### PR TITLE
feat(Dropdown): added secondary styling to split action button

### LINKED ISSUE #7536: Dropdown - add secondary styling option for split button
Adds the option to create a split button dropdown with secondary button styling. Core PR: https://github.com/patternfly/patternfly/pull/4897

### PR DESCRIPTION
**Pull request by andyyvo**
**What**: Closes #7536 . Secondary styling option has been added to `DropdownToggle` component for split action button. An example was created and a unit test/snapshot was added.

**Additional issues**: N/A

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
Looks great. Thanks @andyyvo !

File: packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx [hunk 1]
```diff
@@ -184,6 +184,25 @@ describe('DropdownToggle', () => {
...
+          parentRef={document.createElement('div')}
+        >
+          Dropdown
+        </DropdownToggle>
+      );
+
+      const dropdownToggle = screen.getByRole('button').parentElement;
+
+      expect(dropdownToggle).toHaveClass('pf-m-secondary');
+      expect(asFragment()).toMatchSnapshot();
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx:203]: I'm wondering whether we need this snapshot test. On the one hand it keeps things consistent before this component's tests get updated to our new standard. @wise-king-sullyman what do you think?

**Review by mattnolting — APPROVED** [review]
Nice work @andyyvo! LPTM 🎉

**Review by thatblindgeye — COMMENTED** [review]
This looks great! 🎉 Just had one comment below that I tagged Austin in.

File: packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx (hunk 1 above)
Comment (andyyvo) [reply packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx:203]: That was also the dilemma I had when writing this. All we would need to do is remove the snapshot test part to meet the new standards

**Review by nicolethoen — APPROVED** [review]
After the question for austin is resolved - LGTM!

File: packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx (hunk 1 above)
Comment (wise-king-sullyman) [reply packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx:203]: I would remove the snapshot, I don't think it's really adding value here.

**Review by thatblindgeye — APPROVED** [review]

**Pull request merged**